### DISCUSSION TITLE
Some images does not get drawn

### DISCUSSION THREAD

**Discussion by HudsonSchumaker** [GRRLIB/GRRLIB#12] [Q&A] [answered]
Hi, 
i am using [Kenny Puzzle pack](https://kenney.nl/assets/puzzle-pack)  i can draw the bricks and paddles, but the balls are not being render. Is any kind of specs the png file has to have ?
Thanks in advance.

**Comment by Crayon2000** [comment, marked as answer]
Hello @HudsonSchumaker ,

1. The width and height of the png file must be a multiple of 4
2. The size must be less than 1024px
3. Only 24bit and 32bit PNGs are allowed

**Reply by HudsonSchumaker** [reply]
Thank you very much.
Best regards,
Hudson Schumaker

**Reply by N0NameN0** [reply]
yop there ;)

To be more precise, from what i remember the width and height have to be a power of 2 ;)
Correct me if i'm wrong ;)

Regards,
